universum/modules/api_support.py

```diff
@@ -24,25 +25,25 @@ def __init__(self, api_mode=False, **kwargs):
                     please don't. If you got this message by running it with Universum: something must
                     have gone wrong, may be a bug in Universum itself. Feel free to contact the developers."""))
 
-            with open(os.getenv("UNIVERSUM_DATA_FILE"), "rb+") as data_file:
+            with open(os.getenv("UNIVERSUM_DATA_FILE", ""), "rb+") as data_file:
```

Comment:
This way it won't raise an exception if the env var somehow is absent. And it should.

Reply:
then this code should be rewritten. Relying on 'open' failing when invalid input is given is not a good practice, I think.

Reply:
Sure, but it's a change to be done in a separate PR, isn't it?

Reply:
I think minor changes (like the one needed here) can be done in the same PR.

Reply:
actually, I don't think any change is needed here, since open can't process "" as a filename anyway